trx_examples/streaming/LTE_PA_App/@LTEAppInternals/PlutoRadio.m

```diff
@@ -0,0 +1,170 @@
+function dataRx = PlutoRadio(obj, app, dataTx, frame_ind)
+    if (frame_ind == 1)
+        % filter
+        switch (app.BWDropDown.Value)
+            case '3 MHz'
+                TxFilt = load('lte3_filter_tx.mat');
+            case '5 MHz'
+                TxFilt = load('lte5_filter_tx.mat');
+            case '10 MHz'
+                TxFilt = load('lte10_filter_tx.mat');
+            case '15 MHz'
+                TxFilt = load('lte15_filter_tx.mat');
+            case '20 MHz'
+                TxFilt = load('lte20_filter_tx.mat');
+            otherwise
+                st = dbstack;
+                error("unknown option %s in %s", app.BWDropDown.Value, st.name);
+        end
+        obj.PlutoTx = sdrtx('Pluto', TxFilt.filtnv{:});
+
+        % tx setup
+        obj.PlutoTx.UseCustomFilter = true;
+        obj.PlutoTx.CenterFrequency = app.LOEditField.Value*1e6;
+        obj.PlutoTx.RadioID = app.TxDropDown.Value;
+        obj.PlutoTx.Gain = obj.test_settings.TxGain;
+    end
+    
+    % filter
+    switch (app.BWDropDown.Value)
+        case '3 MHz'
+            RxFilt = load('lte3_filter_rx.mat');
+        case '5 MHz'
+            RxFilt = load('lte5_filter_rx.mat'); 
+        case '10 MHz'
+            RxFilt = load('lte10_filter_rx.mat');
+        case '15 MHz'
+            RxFilt = load('lte15_filter_rx.mat');
+        case '20 MHz'
+            RxFilt = load('lte20_filter_rx.mat');
+        otherwise
+            st = dbstack;
+            error("unknown option %s in %s", app.BWDropDown.Value, st.name);
+    end
+    obj.PlutoRx = sdrrx('Pluto', RxFilt.filtnv{:});
+    
+    % rx setup
+    obj.PlutoRx.UseCustomFilter = true;
+    obj.PlutoRx.CenterFrequency = app.LOEditField.Value*1e6;
+    obj.PlutoRx.RadioID = app.RxDropDown.Value;
+    obj.PlutoRx.NumFramesInBurst = 1;
+    if (frame_ind == 1)
+        app.TxDropDown.Enable = 'off';
+        app.RxDropDown.Enable = 'off';
+    end
+
+    if (app.NumRadios > 1)
+        if strcmp(app.StepOrPlayButton, 'step') || (frame_ind == 1)
+            obj.PlutoRx = FrequencyCorrectionRadios(obj.PlutoTx, obj.PlutoRx);
+            obj.FrequencyCorrection = obj.PlutoRx.FrequencyCorrection;
+            obj.PlutoRx.SamplesPerFrame = 2^23;
+        else
+            obj.PlutoRx.FrequencyCorrection = obj.FrequencyCorrection;
+
+            switch (app.BWDropDown.Value)
+                case '3 MHz'
```

Comment:
Common cases in switch statements should use bracket notation {'3 MHz', '5 MHz', ...}